modules/google-maps/src/google-maps-overlay.ts

```diff
@@ -43,6 +43,11 @@ export default class GoogleMapsOverlay {
 
   /* Public API */
 
+  /** Equivalent of `deck.isInitialized`. */
+  get isInitialized(): boolean {
```

Comment:
It would be nice if the user wouldn't have to worry about this, could we not instead have `pickObject` and friends return null when deck is uninitialized?

Reply:
I want to note that the next planned release is v9.0, so we can make breaking changes if it makes more sense.